racing/db/queries.go

```diff
@@ -13,7 +13,9 @@ func getRaceQueries() map[string]string {
 				name, 
 				number, 
 				visible, 
-				advertised_start_time 
+				advertised_start_time,
+				CASE WHEN strftime('%s', advertised_start_time) >= strftime('%s','now')
+             	THEN 1 ELSE 2 END AS status
```

Comment:
I don't particularly like this solution, but I wanted to:

Calculate the value at the time the model is instantiated if possible, or at least as close to
- Not have to have any "init"y functions required post-instantiation
- I would usually avoid having this kind of business logic in the DB.

Instead I might have the value of `Status` derived from a function/method call. This seems only feasible when the proto models are kept on the edge of the application, and having a translation between some domain model (maybe with a `GetStatus()` method that calculates it) and the proto model. Bit tricky when the proto is being used through all levels of the application including the DB.